third_party/adv_plat/include/bcan_defs.h

```diff
@@ -0,0 +1,56 @@
+/************************************************************************
+*
+* Copyright (c) 2016 Baidu.com, Inc. All Rights Reserved
+*
+************************************************************************/
+
+/**
+* @file bcan_defs.h
+*
+* Baidu ADV (Autonomous Driving Vehicle) bcan basic definitions.
+**/
+
+#ifndef BCAN_DEFS_H
```

Comment:
please use the same convention as other Apollo code files
should be #ifndef THIRD_PARTY_ADV_PLAT_INCLUDE_BCAN_DEFS_H_
also make this change to all other header files